core/math/math_defs.h

```diff
@@ -29,6 +26,9 @@
 /* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
 /**************************************************************************/
+/*                     Work done by Octetdev2 and BSChad.                 */
+/*                       Merging done by Wasabi_Cheetah.                  */
+/**************************************************************************/
```

Comment:
About this,  we can see what the rest of the team says. 

But for references,  this is generally not a good practice to add credits to the preamble,  if we allowed one person to do this everyone would want to do it and then suddenly you have 100 names in the preamble.

Reply:
Co-authors are supposed to be added to the commit message anyway, like what I did with 2d1c65b00d0c320f4f5f010b832ed3ce212446a6 for WhalesState